### PR TITLE
Phidl

### PR DESCRIPTION
**Pull request by joamatab**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/phidl/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+requirements:
+  host:
+    - pip
+    - python >3.6
+  run:
+    - gdspy >=1.5
+    - matplotlib-base
+    - numpy
+    - python
```
Comment (synapticarbors) [inline recipes/phidl/meta.yaml:26]: The same restriction on the python version under `host` should also be used under `run`.

File: recipes/phidl/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/phidl/
+  doc_url: https://phidl.readthedocs.io/en/latest/tutorials.html
+  summary: PHIDL
+  license: MIT
+  license_file: LICENSE
```
Comment (synapticarbors) [inline recipes/phidl/meta.yaml:42]: Add a comment that the license file is manually packaged. I've opened a PR upstream to deal with this in future releases (https://github.com/amccaugh/phidl/pull/86). The comment is a visual reminder that once this is merge, or some other solution is used, the manually packaged copy should be removed:

```
# License file manually packaged. See https://github.com/amccaugh/phidl/pull/86
license_file: LICENSE
...
```

Comment (joamatab) [reply recipes/phidl/meta.yaml:43]: Thank you Joshua

**Review by joamatab — COMMENTED** [review]
thank you Joshua

**Comment by synapticarbors** [comment]
It's not clear to me why when I merged the `pyqtree` recipe a conflict showed up here. Did this branch come off of that one rather than `master`?

**Pull request merged**